repondeur/zam_repondeur/templates/_base_edit.html

```diff
@@ -89,7 +89,15 @@
             content_style: "p, li { line-height: 1.5; margin: 1em 0; }",
             block_formats: "Paragraphe=p;Titre=h3",
             browser_spellcheck: true,
+            init_instance_callback: function (editor) {
+                editor.on('Dirty', function (event) {
+                    var form = document.getElementById("edit-amendement");
+                    var controller = application.getControllerForElementAndIdentifier(form, "unsaved-changes");
+                    controller.formIsChanged();
+                })
+            }
         });
     </script>
+    <script src="{{ request.static_url('zam_repondeur:static/js/unsaved.js') }}"></script>
```

Comment:
I think it would be better to declare it before the tinymce part given it uses it.

I can guess why it works but it might be fragile in the future.